### DISCUSSION TITLE
Wsl download on windows

### DISCUSSION THREAD

**Discussion by Kinglucky04** [Cyfrin/foundry-full-course-cu#2685] [General]
I tried to download the Wsl file for my laptop which is using windows 10 and on installing after the download, my system shutdown and is showing preparing automatic repair whenever I want to boot.

Although I have fix the system but how can I avoid this.

**Comment by EngrPips** [comment]
I have no idea but i trust some giga chad will see this and help.

**Comment by Kinglucky04** [comment]
Alright. Thank you

**Reply by EngrPips** [reply]
You are welcome, Friend.
